datasets/cord-19/cord-19.py

```diff
@@ -0,0 +1,134 @@
+# coding=utf-8
```

Comment:
Can you move the directory from `cord-19` to `cord19` ? We're using snake_case convention. Can you also rename for the dataset script file name accordingly ?